tests/cover/test_reflection.py

```diff
@@ -225,6 +225,10 @@ def test_source_of_lambda_is_pretty():
         lambda x: True
     ) == 'lambda x: True'  # pragma: no cover
 
+def test_bracket_whitespace_is_striped():
+    assert get_pretty_function_description(
+        lambda x: ( x + 1 )
+    ) == 'lambda x: (x + 1)'  # pragma: no cover
```

Comment:
What about a follow up PR that removes them all? It's a mechanical change and if it's in its own commit should make it clearer then spread over many, right?